src/server/vendor/migration/onefoureight/db/pps/pps.pb.go

```diff
@@ -55,7 +55,7 @@ import google_protobuf "github.com/gogo/protobuf/types"
 import google_protobuf1 "github.com/gogo/protobuf/types"
 import google_protobuf2 "github.com/gogo/protobuf/types"
 import _ "github.com/gogo/protobuf/gogoproto"
-import pfs "github.com/pachyderm/pachyderm/src/client/pfs"
```

Comment:
Was this done by manually editing the `pps.proto` file? If so won't this change disappear if you do `make proto`